### PR TITLE
[GOVCMSD8-442] Support for inline media links via Linkit and update to 5.0-beta9 (from 5.0-beta8)

### PR DESCRIPTION
**Pull request by stooit**
## Issue/motivation
Currently there is no way to link to media documents via the WYSIWYG editor without manually copying/pasting direct references to files from the media library.

This is not ideal for two reasons:
- Poor editor experience
- If you change a media entity file then links will break

You can embed media entities directly (e.g via media embed), but they do not render inline (e.g `Text with a <a href="/path/to/file.pdf">linked file</a>`) and are difficult/impossible to coerce into doing so without resorting to theme hackery.

## Resolution
This PR upgrades Linkit to beta9, which has support for core media entities. The bundled config will allow the core linking functionality via WYSIWYG to include media items.

Screen Shot 2019-09-19 at 11 15 07 am

Links will update accordingly if the media entity has it's file replaced/renamed.

I'll submit a follow-up PR with improvements to default document embeds shortly.

### CODE REVIEW

**Review by steveworley — DISMISSED** [review]

**Comment by govindmaloo** [comment]
@steveworley : Please resolve conflict so this can be merged.

**Review by pandaskii — CHANGES_REQUESTED** [review]
Looks good, happy to merge it to distribution after the merge conflicts fixed and test passed.

**Review by pandaskii — APPROVED** [review]

**Comment by pandaskii** [comment]
Looks good, this PR is ready to be merged after testing and approval

thanks

**Pull request merged**
